utils/spid_blob_storage.ts

```diff
@@ -0,0 +1,85 @@
+import { BlobService } from "azure-storage";
+import * as azureStorage from "azure-storage";
+import { Either, either, left, right } from "fp-ts/lib/Either";
+
+export function appendSpidBlob(
```

Comment:
can you add some comments on what this method and the next one are supposed to do ?